app/templates/email_templates/email_response.html

```diff
@@ -12,7 +12,19 @@
         {% block response_file_content %}
         {% endblock %}
     {% else %}
-        {{ content | safe }}
+        {% if content %}
+            {{ content | safe }}
+        {% else %}
+            <p>
+                The {{ agency_name }} has provided additional
+                information about your Freedom of Information Law (FOIL) ID number <a
+                    href='{{ page }}'>{{ request_id }}</a>.
+            </p>
+            <p>
+                Please visit <a href='{{ page }}'>{{ page }}</a> to view additional information and take any necessary
+                action.
+            </p>
+        {% endif %}
```

Comment:
This is necessary because for edit response emails, two templates can be rendered and `content` will be appended to both templates. This change is to allow for `content` to be shown on the requester template if both requester and agency templates are rendered.
Lets assume both the requester and agency users get email indicating a response was edited. The agency user makes changes in the tinymce textarea. The tinymce textarea, along with the changes, are passed in as `content`. `content` will thus be passed into the agency email template when it was just intended for the requester.